### PR TITLE
Updated portal pages for VB getting started

### PR DESCRIPTION
**Pull request by rpetrusha**
# Updated portal pages for VB getting started

## Suggested Reviewers

@mairaw

### CODE REVIEW

File: docs/core/get-started.md [hunk 1]
```diff
@@ -20,8 +20,10 @@ Install .NET Core on [Windows](https://www.microsoft.com/net/core#windows).
 
 You can get started developing .NET Core apps by following these step-by-step tutorials.
 
-* [Building a C# Hello World Application with .NET Core in Visual Studio 2017](./tutorials/with-visual-studio.md) - Learn to to build, debug, and publish a simple .NET Core console application using Visual Studio 2017.
+* [Building a C# Hello World Application with .NET Core in Visual Studio 2017](./tutorials/with-visual-studio.md) - Learn to to build, debug, and publish a simple .NET Core console application using C# and Visual Studio 2017.
 * [Building a class library with C# and .NET Core in Visual Studio 2017](./tutorials/library-with-visual-studio.md) - Learn how to build a class library written in C# using Visual Studio 2017.
+* [Build a Visual Basic Hello World application with .NET Core in Visual Studio 2017](../../core/tutorials/vb-with-visual-studio.md) - Learn to to build, debug, and publish a simple .NET Core console application using Visual Basic and Visual Studio 2017.
```
Comment (mairaw) [inline docs/core/get-started.md]: build is giving you warnings for these new links. Probably should be ./tutorials

**Review by mairaw — CHANGES_REQUESTED** [review]
needs to fix broken links

**Comment by rpetrusha** [comment]
@mairaw, I've fixed them.

**Review by mairaw — APPROVED** [review]
I thought we were going to use tabs for these C#/VB tutorials. But as-is, these changes look good.

**Comment by rpetrusha** [comment]
@mairaw, I merged this, but I hadn't thought of using tabs for the portal page. I think that's a good idea. I'll submit a new PR later.

**Comment by mairaw** [comment]
I didn't mean for the portals but for the actual tutorials. That's why I requested the language tabs.

**Comment by rpetrusha** [comment]
@mairaw, I used tabs for the debugging, publishing, testing, and consuming topics. Tabs didn't seem very helpful for the initial tutorial, so I did separate ones.

**Comment by mairaw** [comment]
Sounds good @rpetrusha. Thanks!

**Pull request merged**